glotaran/parameter/parameter.py

```diff
@@ -42,11 +43,12 @@ def __init__(
         self,
         label: str = None,
         full_label: str = None,
-        expression: str = None,
-        maximum: int | float = np.inf,
-        minimum: int | float = -np.inf,
+        expression: str | None = None,
```

Comment:
Seems label and full_label should have the same type annotation here.

Reply:
Because they are `str`? I don't understand the issue...

Reply:
It is also possible that they are `None` in case of `full_label` it will be set by the setter to `str(None)` which is `'None'` so this is actually right. But `label` can be `str | None` all the time.

Reply:
It's just a minor (nitpicking) thing, but I mean if you make a default values None, the type hint should also include that type, just like with 
`expression: str | None = None,`
it should be
```
label: str | None = None,
full_label: str | None = None,
```